onnxruntime/python/tools/kernel_explorer/kernels/rocm/softmax.cu

```diff
@@ -171,6 +172,37 @@ class CKSoftmax : public IKernelExplorer {
 };
 #endif  // USE_COMPOSABLE_KERNEL
 
+#ifdef USE_TRITON_KERNEL
+template <typename T>
+class SoftmaxTriton : public IKernelExplorer {
```

Comment:
Humm, this doesn't look right. I remember we have a `custom op` mechanism that can register arbitrary kernels. Any concerns on using `custom op`?

Reply:
This is to take advantage of the tuned kernel for AMD Rocm EP.
IkernelExplore will help to select the best performant Kernels from various sources, including triton kernels.